examples/StreamAnalytics/StreamAnalytics.Tests.ps1

```diff
@@ -50,3 +50,220 @@ Describe 'Verify Stream Analytics Cluster' {
     $result | Should -BeInResourceGroup 'rg-test'
   }
 }
+
+Describe 'Stream Analytics Jobs' {
+  it 'Should contain a Stream Analytics Job with the given name' {
+    #arrange
+    $rgName = 'rg-test'
+    $name = 'testjob'
+
+    #act
+    $result = Confirm-AzBPStreamAnalyticsJob -ResourceGroupName $rgName -Name $name
+
+    #assert
+    $result.Success | Should -Be $true
+  }
+
+  it 'Should contain a Stream Analytics Job named testjob' {
+    #arrange
+    $rgName = 'rg-test'
+    $name = 'testjob'
+
+    #act
+    $result = Confirm-AzBPStreamAnalyticsJob -ResourceGroupName $rgName -Name $name
+
+    #assert
+    $result | Should -BeDeployed
+  }
+
+  it 'Should contain a Stream Analytics Job named teststreamcluster in testjob' {
+    #arrange
+    $rgName = 'rg-test'
+    $name = 'testjob'
+
+    #act
+    $result = Confirm-AzBPStreamAnalyticsJob -ResourceGroupName $rgName -Name $name
+    #assert
+    $result | Should -BeInLocation 'westus3'
+  }
+
+  it 'Should be a Stream Analytics Job in a resource group named rg-test' {
+    #arrange
+    $rgName = 'rg-test'
+    $name = 'testjob'
+
+    #act
+    $result = Confirm-AzBPStreamAnalyticsJob -ResourceGroupName $rgName -Name $name
+
+    #assert
+    $result | Should -BeInResourceGroup 'rg-test'
+  }
+
+#######################################################################################################################
```

Comment:
Think we should add this to all the examples. Some of them are really hard to decipher